OpenSim/Examples/Plugins/CoupledBushingForceExample/CoupledBushingForce.h

```diff
@@ -120,73 +102,77 @@ OpenSim_DECLARE_CONCRETE_OBJECT(CoupledBushingForce, Force);
 public:
     // CONSTRUCTION
     CoupledBushingForce();
-    CoupledBushingForce( std::string body1Name, SimTK::Vec3 point1, SimTK::Vec3 orientation1,
-                  std::string body2Name, SimTK::Vec3 point2, SimTK::Vec3 orientation2,
-                  SimTK::Mat66 stiffnessMat, SimTK::Mat66 dampingMat);
-    CoupledBushingForce(const CoupledBushingForce &aForce);
-    virtual ~CoupledBushingForce();
+    /** Construct the CoupleBushingForce given the two offset frames to which
+    it attaches by name. Stiffness and damping properties are supplied by 6x6
+    matrices. Rotational stiffness (damping) in N/rad(/s) and translational in
+    N/m(/s). Off-diagonals represent the coupling terms.
+    See property declarations for more information. **/
+    CoupledBushingForce(const std::string& frame1Name,
+                            SimTK::Vec3 point1, SimTK::Vec3 orientation1,
+                        const std::string& body2Name,
+                            SimTK::Vec3 point2, SimTK::Vec3 orientation2,
+                         SimTK::Mat66 stiffnessMat, SimTK::Mat66 dampingMat);
 
-    CoupledBushingForce& operator=(const CoupledBushingForce &aForce);
-    void copyData(const CoupledBushingForce &aForce);
+    virtual ~CoupledBushingForce();
```

Comment:
Better to write `~CoupledBushingForce() override;` which will fail to compile if the base class doesn't have a virtual destructor (which it should).